spellbook/models/nft/ethereum/nft_ethereum_schema.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
```

Comment:
maybe a misunderstanding of how dbt is setup, but why wouldn't this fit into the other schema file?
https://github.com/duneanalytics/abstractions/blob/master/spellbook/models/nft/nft_schema.yml